test/roundtrip/invalid-name.txt

```diff
@@ -0,0 +1,13 @@
+;;; TOOL: run-roundtrip
+;;; ARGS: --stdout --no-check --enable-annotations --debug-names
+(module
+  (type (;0;) (func (result i32)))
+  (func (@name "$hi hello hey") (type 0) (result i32)
+    i32.const 42))
+(;; STDOUT ;;;
+(module
+  (type (;0;) (func (result i32)))
+  (func (@name "$hi hello hey") (type 0) (result i32)
+    i32.const 42)
+  (@custom "name" "\01\0f\01\00\0chi hello hey\02\03\01\00\00"))
```

Comment:
Do we want to be printing both the `@name` annotation and also the `@custom "name"` annotation with the same info? (Not 100% sure what the right behavior is, but we do suppress the `@custom` annotation for most other cases that are already handled elsewhere...)

Reply:
Hmm, this is the current behavior of names in general. Even without the `@name` annotation, both are included. I'll work on suppressing this, though.